clients/python/tests/REST_bindings_test.py

```diff
@@ -20,8 +20,13 @@
 
 @pytest.fixture
 @cleanup
-async def client() -> AsyncIterator[ModelRegistryServiceApi]:
-    config = mr_openapi.Configuration(REGISTRY_URL)
+async def client(user_token: str, verify_ssl: bool) -> AsyncIterator[ModelRegistryServiceApi]:
+    params = {}
+    if user_token:
+        params["access_token"] = user_token
+    if verify_ssl is not None:
+        params["verify_ssl"] = verify_ssl
```

Comment:
not nitpicky on this one, but you can always set these params directly in the dict even if its None, because they are still going to default to `None` in the function param since they are optional?

```suggestion
params = {"verify_ssl": verify_ssl, "access_token": user_token}
```